introduction.md

```diff
@@ -3,5 +3,9 @@ My name is iaan.
 I'm father of two sons. And really into sports. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?